### PR TITLE
Adding FS TC CEPH-11300

### PR DESCRIPTION
**Pull request by AmarnatReddy**
Adding FS TC CEPH-11300
CEPH-11300: Running basic bash commands on fuse,Kernel and NFS mounts

    Steps Performed:
    1. Create FS and mount on all the ways (Fuse, kernel, NFS)
    2. Run directory_cmds, file_cmds, search_cmds
    
IBM Jenkins log : https://159.23.92.24/job/Custom%20Test%20Run/16/console
PSI logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4caro/

Single test pass log : [http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run- 9489NZ](http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ha80o/)
Signed-off-by: Amarnath K <amk@amk.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/fs_basic_bash_cmds.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    Running commands on mounted directory
+    Args:
+        clients:
+        mounting_dir:
+        filename:
+
+    Returns:
+
+    """
+    log.info("Basic directory operations")
```
Comment (hkadam134) [inline tests/cephfs/fs_basic_bash_cmds.py:156]: Can we also include some more operations like : 
ls, cp, mv, rm, ln, more, less,  chmod. 

And these operations should be performed on a directory with a large amount of data or a huge number of small files.

File: tests/cephfs/fs_basic_bash_cmds.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+        log.info("---clean up---------")
+        # fs_util.client_clean_up(
+        #     "umount", fuse_clients=[clients[0]], mounting_dir=fuse_mounting_dir
+        # )
+        # fs_util.client_clean_up(
+        #     "umount",
+        #     kernel_clients=[clients[0]],
+        #     mounting_dir=kernel_mounting_dir,
+        # )
+
```
Comment (yogesh-mane) [inline tests/cephfs/fs_basic_bash_cmds.py:149]: Could we correct this?

**Review by hkadam134 — COMMENTED** [review]
"ln" is still missing Amar,

Can you include soft links and hard links also?

**Review by hkadam134 — APPROVED** [review]
LGTM

**Review by hkadam134 — APPROVED** [review]
LGTM

**Pull request merged**